setup.py

```diff
@@ -2,6 +2,7 @@
 from setuptools.command.test import test as TestCommand
 import sys
 
+from ducktape.utils.util import ducktape_version
```

Comment:
Hmm, are you sure this actually works properly? Do you know what the rules are for how PYTHONPATH is setup when you use setup.py directly, via pip, etc? I'm not sure, so I'm not sure we're guaranteed it will pick up `ducktape.utils.util` from the directory relative to this file or if it's possible it can grab, for example, an installed version instead. (See previous note about how requests handles this -- not sure that approach is ideal either, but they probably know how to handle this better than I do.)